R/R.r

```diff
@@ -9,7 +9,12 @@ R <- function(options, path = tempdir(), env_vars = NULL, ...) {
     on.exit(set_path(old))
   }
 
-  in_dir(path, system_check(r_path, options, c(r_env_vars(), env_vars), ...))
+  tmp_user_profile <- tempfile("Rprofile")
+  writeLines(text = 'options(repos=c(CRAN="http://cran.rstudio.com"))',
+             con = tmp_user_profile)
+
+  in_dir(path, system_check(r_path, options, c(R_PROFILE_USER=tmp_user_profile,
```

Comment:
Should `r_env_vars()` be responsible for creating the file, too? A faint smell...